modules/shared/components/molecules/UserInfo/UserInfo.test.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import UserInfo from './Index';
+
+describe('UserInfo', () => {
+  test('if variant equalls annonymous, name becomes Anonymous as well', () => {
+    render(<UserInfo size="medium" variant="anonymous" date="2 hours" />);
+    expect(screen.getByTestId('name')).toHaveTextContent('Anonymous');
+  });
+
+  test('if variant equalls annonymous, name becomes Anonymous as well even if a name is set', () => {
+    render(
+      <UserInfo
+        size="medium"
+        variant="anonymous"
+        date="2 hours"
+        name="Ahmed Ayoub"
+      />,
+    );
+    expect(screen.getByTestId('name')).toHaveTextContent('Anonymous');
+  });
+
+  test('if variant equalls filled, name is whatever passed to the name prop', () => {
+    render(
+      <UserInfo
+        size="medium"
+        variant="filled"
+        date="2 hours"
+        name="Ahmed Ayoub"
+      />,
+    );
+    expect(screen.getByTestId('name')).toHaveTextContent('Ahmed Ayoub');
+  });
+
+  test('if variant equalls notFilled, name is whatever passed to the name prop', () => {
```

Comment:
which I did. My point was to make sure that the user has it's name whether the image filled or not.
but when the image / user is an anonymous don't give him a name.

Reply:
what is the difference between the last two tests?

Reply:
the last two tests have the same goal which's name === "Ahmed Ayoub". but one for variant filled and the other for notFilled.

Reply:
cool. for some reason (maybe because I am from mobile) I saw the two variants as `filled`. now I can see that they are different and the typo was addressed 👍